frontend/device/pocketbook/device.lua

```diff
@@ -506,6 +506,16 @@ local PocketBook617 = PocketBook:extend{
     hasNaturalLight = yes,
 }
 
+-- PocketBook Basic Lux 4 (618)
+local PocketBook618 = PocketBook:extend{
+    model = "PB618",
```

Comment:
@Frenzie is there a preference here on how to name the model id?

For the lux 3 this was `PBBLux3`. I can imagine that makes it easier to figure out what model were talking about without cross referencing.

Reply:
Yes, something like PBLux4, see #4479.